### ISSUE TITLE
Need preprocessing model

### ISSUE THREAD

**Issue by qinxin99** [Wenxuan-1119/TransBTS#34]
Hello, would you mand send me a preprocessing model, 
qinxinni99@gmail.com

**Comment by 111Kaushal** [comment]
Hello all!
We trained the model but only for 1000 epochs. You can download and find more details here: - https://github.com/111Kaushal/Testing-and-validation-of-TransBTSV2

**Comment by Taocunguo** [comment]
> Hello all! We trained the model but only for 1000 epochs. You can download and find more details here: - https://github.com/111Kaushal/Testing-and-validation-of-TransBTSV2

hello,I used your preprocessing model on the validation set of BraTS2020, but got poor results, what are the results you got?
